### DISCUSSION TITLE
New Project Lead

### DISCUSSION THREAD

**Discussion by cseickel** [nvim-neo-tree/neo-tree.nvim#1467] [Announcements]
Hi everyone,

I just wanted to let you know that I'm stepping down as the lead developer of Neo-tree and passing the reigns on to @pysan3. @pysan3 has a lot of great ideas and plans for taking Neo-tree into v4 with a cleanup of all async code and inclusion of https://github.com/pysan3/pathlib.nvim to handle all of the cross platform path nastiness that causes us so much problems.

I'm not leaving Neovim or anything, I still love Neovim and I will continue to be a Neo-tree user and probably a part-time contributor. I just don't have it in me to lead the project anymore. The reason is just that I'm in a different place than I was when I started the project, and right now, I just don't have the capacity for this work.

Thank you @pysan3!

**Comment by oldbop** [comment]
Thank you for your great work! All the best! 🙋‍♂️
